src/OrchardCore.Modules/OrchardCore.Queries/Sql/SqlQuerySource.cs

```diff
@@ -39,26 +43,40 @@ public class SqlQuerySource : IQuerySource
             _templateOptions = templateOptions.Value;
         }
 
-        public string Name => "Sql";
+        public string Name
+            => SourceName;
```

Comment:
```suggestion
        public string Name  => SourceName;
```

Reply:
I prefer the new line for consistency.

Reply:
Consistency with what, exactly? The point of this syntax is that it can be compact. The line break is unnecessary, especially for a property.

Reply:
Once of the nice advantage of the body expression is that we can use vertical space and try to reduce horizontal code. Eveytime I created body expression, I start it on a new line for this reason "everywhere else I committed code". So for consistency with the rest of the code in main, I place these to on new line.

Reply:
Let's defer this discussion to when we have analyzers to guard consistency.